Automattic/wp-calypso
# Migrate images in PremiumPlanDetails to be managed by webpack

That creates immutable names with a hash suffix for the image assets. Is a blueprint how to fix issues like #34634 or #34641. The URL will change if the source image changes.

Cc: @sirreal and @ockham who worked on the webpack file loader some time ago: opinions on the `client/assets/images` path for the webpack-managed assets? There's also an option to put the images in the same directory with the components that use them.

That doesn't work for `google-adwords.svg` though, which is used at 3 different places across the app.

**How to test:**
For a site with purchased Premium plan, go to
```
/checkout/thank-you/:site_slug/:receipt_id
```

Check that all images on the Thank You page are loaded, and verify in Devtools that the URLs are hashed:

<img width="741" alt="Screenshot 2019-07-19 at 15 55 18" src="https://user-images.githubusercontent.com/664258/61540790-a38afd00-aa3e-11e9-8ad0-dd96c40294f6.png">
